docs/UserGuide.md

```diff
@@ -280,7 +280,7 @@ If a required parameter is missing (e.g., NAME, EMAIL)
 
   ![add format](images/add-UG/addmissingparam.png)
 
-If a parameter is provided in an invalid format (e.g., an invalid email address), an error message should indicate the invalid format.
+If a parameter you provided in an invalid format (e.g., an invalid email address), an error message should indicate the invalid format.
```

Comment:
add an 'is' after provided (i.e. "If a parameter you provided **_is_** in an invalid format..."

Reply:
Thank you, Bhanuka!

Reply:
LGTM!